packages/hooks/useFundingRounds.js

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react'
+import { useQuery } from 'urql'
+import groupFundingRoundsQuery from '@hylo/graphql/queries/groupFundingRoundsQuery'
+
+// Helper to present funding round data
+function presentFundingRound (fundingRound) {
```

Comment:
So there are two, slightly different presentFundingRound functions, this one and the one in useFundingRound